src/Pages/ApiResourceDetails.razor

```diff
@@ -43,7 +43,7 @@ else
                 </div>
             </div>
             <div class="tab-pane fade show" id="secrets-tab-pane" role="tabpanel" aria-labelledby="secrets-tab" tabindex="1">
-                <QuickGrid Items="@SelectedApiResource.Secrets.AsQueryable()" Class="table">
+                <QuickGrid Items="@SelectedApiResource.Secrets.AsQueryable()" Class="table"Pagination="@_paginationSecrets">
```

Comment:
Misschien even een spatie tussen "table" en Pagination?

Reply:
Added the space in ec7e71a